Core/Base/AgentModule.py

```diff
@@ -337,6 +347,11 @@ def am_go( self ):
       cD = self.__moduleProperties[ 'cyclesDone' ]
       self.log.notice( "Remaining %s of %s cycles" % ( mD - cD, mD ) )
     self.log.notice( "-"*40 )
+    # use SIGALARM as a watchdog interrupt if enabled
+    watchdogInt = self.am_getWatchdogTime()
```

Comment:
Should it be casted to int ? I think this is a string returned ( if not default )